tests/ui/limits/issue-56762.stderr

```diff
@@ -1,14 +1,14 @@
-error[E0080]: values of the type `[u8; 2305843009213693951]` are too big for the current architecture
+error[E0080]: could not evaluate static initializer
```

Comment:
Now that this is always a hard error I don't see a good reason to emit a different diagnostic just for the "too big" case.